packages/upstream-extra/fileutils.0.5.1/findlib

```diff
@@ -1 +0,0 @@
-fileutils
```

Comment:
I was not sure what purpose they still serve, but looks like OPAM2 no longer has code to deal with them and they got dropped from upstream repo as well so its ok: https://github.com/ocaml/opam-repository/pull/9328

Reply:
Also the current `opam` has not had code for them since years